cvat/apps/engine/view_utils.py

```diff
@@ -92,3 +95,37 @@ def decorator(f):
         return f
 
     return decorator
+
+def get_410_response_for_export_api(path: str) -> HttpResponseGone:
+    return HttpResponseGone(textwrap.dedent(f"""\
+        This endpoint is no longer supported.
+        To initiate the export process, use POST {path}.
+        To check the process status, use GET /api/requests/rq_id,
+        where rq_id is obtained from the response of the previous request.
+        To download the prepared file, use the result_url obtained from the response of the previous request.
+    """))
+
+def get_410_response_when_checking_process_status(process_type: str, /) -> HttpResponseGone:
+    return HttpResponseGone(textwrap.dedent(f"""\
+        This endpoint no longer supports checking the status of the {process_type} process.
+        The common requests API should be used instead: GET /api/requests/rq_id,
+        where rq_id is obtained from the response of the initializing request.
+    """))
+
+class DeprecatedResponse(Response):
+    def __init__(self,
+        data=None,
+        status=None,
+        template_name=None,
+        headers=None,
+        exception=False,
+        content_type=None,
+        *,
+        deprecation_date: datetime,
+    ):
+        headers = headers or {}
+        # https://greenbytes.de/tech/webdav/draft-ietf-httpapi-deprecation-header-latest.html#the-deprecation-http-response-header-field
```

Comment:
This draft has been published as [RFC 9745](https://www.rfc-editor.org/rfc/rfc9745.html).